config/install/globalredirect.settings.yml

```diff
@@ -1,9 +0,0 @@
-deslash: true
-nonclean_to_clean: true
-access_check: false
-normalize_aliases: true
-canonical: false
```

Comment:
this is not implemented. Create an issue to re-implement it: "Port canonical link feature"